internal/goutils/utils.go

```diff
@@ -0,0 +1,27 @@
+package goutils
+
+// Ptr returns a pointer to the given value.
+func Ptr[T any](input T) *T {
```

Comment:
Can you actually spell this out and call this function `Pointer` for clarity?

For context: https://google.github.io/styleguide/go/decisions#variable-names
- refer to the bullet that says "Do not simply drop letters to save typing. For example Sandbox is preferred over Sbx, particularly for exported names."

Reply:
In my coding practice I try not to do this, the reason it was shortned is that I relied on one of the posts and for some reason the shortened version didn't trigger me.

In general I prefer longer function names and variables so that it becomes sort of an abstract English text, "cooking recipe instruction" ahaha.